src/content/en/2020/seo.md

```diff
@@ -83,183 +84,227 @@ Response Code | All Devices | Mobile | Desktop
-  )
-}}
+) }}
 
 The median mobile site also displays 11.5% more words when rendered than its raw HTML counterpart.
 
-Across our sample set, there are disparities across the combination of mobile/desktop and rendered/non-rendered. This suggests that although search engines are continually improving in this area, most sites across the web are missing out on opportunities to improve their Organic Search visibility through a stronger focus on ensuring their content is available and indexable. This is also an issue of concern because the lion’s share of available SEO tools do not crawl in the above combination of contexts and automatically identify this as an issue.
+{# TODO(authors): Same question about "Organic Search" capitalization. #}
+Across our sample set, there are disparities across the combination of mobile/desktop and rendered/non-rendered. This suggests that although search engines are continually improving in this area, most sites across the web are missing out on opportunities to improve their Organic Search visibility through a stronger focus on ensuring their content is available and indexable. This is also a concern because the lion's share of available SEO tools do not crawl in the above combination of contexts and automatically identify this as an issue.
 
-This year, the median desktop home page was found to have 402 words and the mobile 348 words. While [last year](../2019/seo#word-count), the median desktop home page had 346 words, and the median mobile home page had a slightly lower word count at 306 words. This represents 16.2% and 13.7% growth respectively.
+This year, the median desktop home page was found to have 402 words and the mobile page had 348 words. While [last year](../2019/seo#word-count), the median desktop home page had 346 words, and the median mobile home page had a slightly lower word count at 306 words. This represents 16.2% and 13.7% growth respectively.
 
 #### Headings
 
-Heading elements (h1-h6) act as a mechanism to visually indicate structure in a page’s content. Although these HTML elements don’t carry the weight they used to in search rankings, they still act as a valuable way to structure pages and signal other elements in the SERPs like Featured Snippets or other extraction methods that align with [Google’s new passage indexing](https://www.blog.google/products/search/search-on/).
+Heading elements (`H1`-`H6`) act as a mechanism to visually indicate structure in a page's content. Although these HTML elements don't carry the weight they used to in search rankings, they still act as a valuable way to structure pages and signal other elements in the search engine results pages (SERPs) like _featured snippets_ or other extraction methods that align with [Google's new passage indexing](https://www.blog.google/products/search/search-on/).
 
+{# TODO(analysts, authors): Nit: replace "This includes empty ones" with "Heading levels" and clarify including empty headings in the prose. #}
```

Comment:
@Tiggerito @aleyda

Reply:
@rviscomi added the clarification in the text cc @Tiggerito

Reply:
301 to @antoineeripret (I hope you are the chart expert person)

Reply:
@Tiggerito : Done and PNG file added to the commit.